### PR TITLE
feat: add terms, disclaimer and privacy policy pages

### PR DESCRIPTION
**Pull request by spaenleh**
In this PR I add the terms page:

terms_page

Disclaimer page:

disclaimer_page

And the privacy policy page:

privacy_page

### CODE REVIEW

File: src/modules/landing/footer/Footer.tsx [hunk 1]
```diff
@@ -81,7 +81,18 @@ const socialLinks = [
   },
 ];
 
-const internalLinkActiveProp = () => ({ sx: { textDecoration: 'underline' } });
+const internalLinkActiveProp = () => ({
+  sx: {
+    backgroundColor: '#00000040',
+    // textDecoration: 'underline',
```
Comment (pyphilia) [inline src/modules/landing/footer/Footer.tsx:87]: Remove?

File: src/modules/landing/privacyPolicy/Layout.tsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
+import { ReactNode } from 'react';
```
Comment (pyphilia) [inline src/modules/landing/privacyPolicy/Layout.tsx:1]: `Layout.tsx` but it contains many components? Should it be `layouts.tsx`? Or `components.tsx`?

File: src/routes/_landing/policy.tsx [hunk 3]
```diff
@@ -0,0 +1,328 @@
...
+    <Section title={t('TITLE')}>
+      <Paragraphs>
+        <Typography>{t('DEFINITION')}</Typography>
+        <Typography>{t('COMPLIANCE')}</Typography>
+        <Typography>
+          <Trans
+            t={t}
+            i18nKey="CONTACT"
+            values={{ email: PRIVACY_EMAIL }}
+            components={{ mail: <a href={`mailto:${PRIVACY_EMAIL}`}>_</a> }}
```
Comment (pyphilia) [inline src/routes/_landing/policy.tsx:61]: _ ?

**Review by pyphilia — APPROVED** [review]

**Review by ReidyT — APPROVED** [review]
I agree with @pyphilia, otherwise it looks really good 😆

File: src/modules/landing/privacyPolicy/Layout.tsx (hunk 2 above)
Comment (spaenleh) [reply src/modules/landing/privacyPolicy/Layout.tsx:1]: I will change the file name to `layouts.tsx`

**Comment by spaenleh** [comment]
@pyphilia @ReidyT Thank you for your reviews, I have fixed the issues and will merge once the tests pass ! 👍

**Pull request merged**